src/main/kotlin/org/sourcegrade/jagr/Main.kt

```diff
@@ -42,14 +42,15 @@ class MainCommand : CliktCommand() {
    */
   private val child by option("--child", "-c").flag()
   private val exportOnly by option("--export-only", "-e").flag()
+  private val rainbow by option("--rainbow", "-r").flag()
```

Comment:
Don't add the "-r" option, that should probably be reserved for something else in the future.